### DISCUSSION TITLE
Inertia,js : can not upload moderaly large photos

### DISCUSSION THREAD

**Discussion by begueradj** [inertiajs/inertia#1757] [Help]
In my Laravel Inertia Vue application, I can upload photos of less than 2 MB like this:

```js
 form.post(route('demands.store'), {});  
```

But I can not post images larger than that.

My php.ini settings are enough for the image sizes I have (none of them exceeds 6 MB):
```php
upload_max_filesize = 1G
post_max_size = 3G
memory_limit = 5G
```

Any reason why ?

**Comment by T-Huwa** [comment]
Maybe if you tried using M instead of G
